### PR TITLE
throwing error on unencrypted proxy pass

### PR DESCRIPTION
**Pull request by herman-kailey**
fix for missing error message when installing via proxy without an encrypted password

### CODE REVIEW

**Comment by herman-kailey** [comment]
fixes #10945

File: dev/com.ibm.ws.install/src/com/ibm/ws/install/RepositoryConfigUtils.java [hunk 1]
```diff
@@ -203,7 +203,8 @@ public static RestRepositoryConnectionProxy getProxyInfo(Properties repoProperti
                     } catch (InvalidPasswordDecodingException ipde) {
                         decodedPwd = proxyPwd;
                         logger.log(Level.FINE, Messages.INSTALL_KERNEL_MESSAGES.getLogMessage("LOG_PASSWORD_NOT_ENCODED_PROXY", proxyURL) + InstallUtils.NEWLINE);
-                        setProxyAuthenticator(proxyHost, proxyPort, proxyUser, decodedPwd);
+                        throw new InstallException(Messages.INSTALL_KERNEL_MESSAGES.getLogMessage("ERROR_TOOL_PROXY_PWD_NOT_ENCRYPTED"));
+                        //setProxyAuthenticator(proxyHost, proxyPort, proxyUser, decodedPwd);
```
Comment (alvinso) [inline dev/com.ibm.ws.install/src/com/ibm/ws/install/RepositoryConfigUtils.java]: remove setProxyAuthenticator if we don't need it.

File: dev/com.ibm.ws.install/src/com/ibm/ws/install/RepositoryConfigUtils.java [hunk 2]
```diff
@@ -400,7 +401,7 @@ public static boolean isWlpRepoEnabled(Properties repoProperties) {
      * Gets the name of the repository by connection and properties
      *
      * @param repoProperties The repository properties
-     * @param repoConn The repository connection
+     * @param repoConn       The repository connection
```
Comment (alvinso) [inline dev/com.ibm.ws.install/src/com/ibm/ws/install/RepositoryConfigUtils.java]: Fix the white space

File: dev/com.ibm.ws.install/src/com/ibm/ws/install/RepositoryConfigUtils.java [hunk 3]
```diff
@@ -440,7 +441,7 @@ public static String getRepoName(Properties repoProperties, RepositoryConnection
     /**
      * Checks if the inputed Repository Connection is a liberty repository
      *
-     * @param lie The RestRepositoryConnection connection
+     * @param lie            The RestRepositoryConnection connection
```
Comment (alvinso) [inline dev/com.ibm.ws.install/src/com/ibm/ws/install/RepositoryConfigUtils.java]: Fix the white space

**Pull request closed without merging**